src/app/contact-us/page.jsx

```
@@ -52,6 +71,29 @@ const ContactUs = () => {
           </>
         )}
         <div className={styles.borderView} />
+        {/* --- OTC Card --- */}
+        <div style={{display: 'flex', justifyContent: 'center', marginTop: 30}}>
+          <button className={styles.cardBox} onClick={handleOTCClick}>
+            <div className={styles.cardItemContainer}>
+              <Image
+                src={
+                  getAppAssets()?.['buy_card_2']?.[themeType] || cards[1].src
+                }
+                alt="OTC"
```

ESLint

[error] Replace `"OTC"` with `'OTC'`

(prettier/prettier)